.travis.yml

```diff
@@ -1,3 +1,7 @@
+sudo: required
```

Comment:
Reference the travis issue? w/ a TODO to remove?

Reply:
Done.